### PR TITLE
New workflow

### PR DESCRIPTION
**Pull request by paulgrever**

### CODE REVIEW

**Comment by paulgrever** [comment]
@spencern  ready to move into dev

File: client/templates/dashboard/dashboard.html [hunk 1]
```diff
@@ -2,10 +2,105 @@
...
+            <th>Packing</th>
+            <th>Packed</th>
+            <th>Ready to Ship</th>
+            <th>Shipped</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Yesterday</td>
+            <td>{{yesterdaysOrders.orderCreated}}</td>
```
Comment (spencern) [inline client/templates/dashboard/dashboard.html:49]: We can dry this up a bit by looping through the relevant statuses and days that we are interested in.

File: server/methods/orderDetail.js [hunk 2]
```diff
@@ -170,41 +173,30 @@ Meteor.methods({
       }
     });
   },
-  'advancedFulfillment/orderCompleted': function (order, userId) {
+  'advancedFulfillment/orderCompletionVerifier': function (order, userId) {
```
Comment (spencern) [inline server/methods/orderDetail.js:176]: Needs security.

File: server/methods/orderDetail.js [hunk 3]
```diff
@@ -240,5 +232,47 @@ Meteor.methods({
     ReactionCore.Collections.Orders.update({_id: orderId}, {
       $set: {orderNotes: orderNotes}
     });
+  },
+  'advancedFulfillment/printInvoices': function (startDate, endDate, userId) {
```
Comment (spencern) [inline server/methods/orderDetail.js:236]: Needs Security

**Pull request merged**